nucypher/blockchain/eth/sol/source/contracts/MinersEscrow.sol

```diff
@@ -305,7 +309,41 @@ contract MinersEscrow is Issuer {
         }
     }
 
+    /**
+    * @notice Checks if `reStake` parameter is available for changing
+    * @param _miner Miner
+    **/
+    function isReStakeLocked(address _miner) public view returns (bool) {
```

Comment:
This function is just view so nothing wrong to call it even before initialization. And adding this modifier increases (a little bit) size of contract

Reply:
I was thinking about this as defense in depth measure, since the stake concept doesn't make sense in uninitialized accounts. Anyway, I see your point and I'm OK if we leave it as it is.

Reply:
One moment more, contract will be not initialized only few moments, so protection only in critical functions to minimize logic and contract size